src/UiSlider.vue

```diff
@@ -263,23 +258,15 @@ export default {
             };
         },
 
-        refreshSize() {
-            this.thumbSize = this.$refs.thumb.offsetWidth;
-            this.trackLength = this.$refs.track.offsetWidth;
-            this.trackOffset = this.getTrackOffset(this.$refs.track);
-        },
-
         initializeSlider() {
             document.addEventListener('touchend', this.onDragStop);
             document.addEventListener('mouseup', this.onDragStop);
             document.addEventListener('click', this.onExternalClick);
 
             this.$on('window-resize', () => {
```

Comment:
We can now remove the listener as we were stopping the drag due to the possible size change. The `RespondsToWindowResize` mixin below can also be removed (with the import above as well)